Orange/widgets/model/owneuralnetwork.py

```diff
@@ -184,10 +190,10 @@ def get_learner_parameters(self):
                 ("Replicable training", self.replicable))
 
     def get_hidden_layers(self):
+        self.Warning.no_layers.clear()
         layers = tuple(map(int, re.findall(r'\d+', self.hidden_layers_input)))
         if not layers:
-            layers = (10,)
-            self.hidden_layers_input = "10,"
+            self.Warning.no_layers()
```

Comment:
This is OK, but you could also skip clearing and do `self.Warning.no_layers(shown=not layers)`.